rcl/src/rcl/dynamic_message_type_support.c

```diff
@@ -23,23 +23,26 @@ extern "C"
 
 #include "rmw/dynamic_message_type_support.h"
 
+#include "rcl/allocator.h"
 #include "rcl/common.h"
 #include "rcl/error_handling.h"
 #include "rcl/dynamic_message_type_support.h"
 #include "rcl/type_hash.h"
 #include "rcl/types.h"
 
 
-/// Create a rosidl_message_type_support_t from a TypeDescription message
+/// Initialize a rosidl_message_type_support_t from a TypeDescription message
 RCL_PUBLIC
 RCL_WARN_UNUSED
 rcl_ret_t
-rcl_dynamic_message_type_support_handle_create(
+rcl_dynamic_message_type_support_handle_init(
   const char * serialization_lib_name,
   const rosidl_runtime_c__type_description__TypeDescription * description,
-  rosidl_message_type_support_t ** ts)
+  rcl_allocator_t * allocator,
+  rosidl_message_type_support_t * ts)
 {
   RCUTILS_CHECK_ARGUMENT_FOR_NULL(ts, RCUTILS_RET_INVALID_ARGUMENT);
+  RCUTILS_CHECK_ARGUMENT_FOR_NULL(allocator, RCUTILS_RET_INVALID_ARGUMENT);
```

Comment:
You maybe should also check 

```
  if (!rcutils_allocator_is_valid(&allocator)) {
```

Reply:
I might defer this so we don't invalidate the windows debug CI run :x

Reply:
You are going to have to re-run it anyway, since you changed all of the function signatures for `rmw_init_serialization_support`